activerecord/lib/active_record/relation/calculations.rb

```diff
@@ -184,6 +177,27 @@ def ids
 
     private
 
+      def _pluck(column_names, unsafe_raw)
```

Comment:
Is the `_` prefix okay here, or should I choose something more descriptive like `#pluck_internal`?: